### ISSUE TITLE
[Filter Script] skill_level() does not exist

### ISSUE THREAD

**Issue by Davidj361** [splintermind/Dwarf-Therapist#246]
![A screenshot](http://i.imgur.com/nNvYGjM.png)

The script editor is complaining that skill_level is not a method of 'd'.

Same issue with these other functions:
- float skill_level_precise() **returns the interpolated skill level (eg. 4.7)
- float skill_level_raw() **returns the raw skill level, which can exceed  20 (eg. 120)
- float skill_level_raw_precise() **returns the interpolated raw skill level (eg. 120.9)

But these ones work:
- int rust_level()**returns the worst level of rust found in the unit's skills, higher indicating more rust
- int total_skill_levels()

I'm currently using Dwarf Therapist 31.0.0 that was compiled with Qt 5.1.1, running with Qt 5.1.1

**Comment by Davidj361** [comment]
Found the issue
**Dwarf-Therapist-DF2014\inc\dwarf.h line 300**

``` c++
    //! returns the numeric rating for the this dwarf in the skill specified by skill_id
    float get_skill_level(int skill_id, bool raw = false, bool precise = false);
    //! convenience functions for skill level
    float skill_level(int skill_id);
    float skill_level_raw(int skill_id);
    float skill_level_precise(int skill_id);
    float skill_level_raw_precise(int skill_id);
```

These functions do not start with **Q_INVOKABLE** like the other scripting functions. Another thing I noticed is that the documentation does not correspond to what the function parameters are.
![screenshot](https://i.imgur.com/6dv1jaY.png)
